Packs/ArcherRSA/Integrations/ArcherV2/ArcherV2.py

```diff
@@ -288,7 +290,7 @@ def update_session(self):
             'Password': self.password
         }
         try:
-            res = self._http_request('POST', '/api/core/security/login', json_data=body)
+            res = self._http_request('POST', f'/{API_ENDPOINT}/core/security/login', json_data=body)
```

Comment:
add the API endpoint to the base URL in the main function instead of applying it each time